src/Controls/src/Build.Tasks/nuget/buildTransitive/netstandard2.0/Microsoft.Maui.Controls.targets

```diff
@@ -21,6 +21,15 @@
 		<Analyzer Include="$(MSBuildThisFileDirectory)Microsoft.Maui.Controls.SourceGen.dll" IsImplicitlyDefined="true" />
 	</ItemGroup>
 
+	<!-- Enable BindingSourceGen -->
+	<PropertyGroup>
+		<_MauiEnableBindingInterceptors Condition=" '$(_MauiEnableBindingInterceptors)' == '' and '$(DisableMauiAnalyzers)' != 'true' ">true</_MauiEnableBindingInterceptors>
```

Comment:
I think it makes sense to have a private MSBuild property to turn it off. 👍 Customers can opt out of the new source generator with `_MauiEnableBindingInterceptors=false` if something goes wrong with their build.